lib/src/sai_redis_generic_get_stats.cpp

```diff
@@ -0,0 +1,192 @@
+#include "sai_redis.h"
+#include "meta/saiserialize.h"
+
+sai_status_t internal_redis_get_stats_process(
+        _In_ sai_object_type_t object_type,
+        _In_ uint32_t count,
+        _Out_ uint64_t *counter_list,
+        _In_ swss::KeyOpFieldsValuesTuple &kco)
+{
+    SWSS_LOG_ENTER();
+
+    // key:         sai_status
+    // field:       stat_id
+    // value:       stat_value
+
+    const std::string &key = kfvKey(kco);
+    const std::vector<swss::FieldValueTuple> &values = kfvFieldsValues(kco);
+
+    std::string str_sai_status = key;
+
+    sai_status_t status;
+
+    sai_deserialize_status(str_sai_status, status);
+
+    if (status == SAI_STATUS_SUCCESS)
+    {
+        uint32_t i = 0;
+        for (const auto &v : values)
+        {
+            uint64_t value = 0;
+
+            value = stoull(fvValue(v));
+            counter_list[i] = value;
```

Comment:
>i [](start = 25, length = 1)

check for index out of range at the loop beginning?

Reply:
Done